src/templates/_common/_layout/footer.jsx

```diff
@@ -136,7 +136,7 @@ const Footer = () => (
                                     { media: 'youtube',   href: 'https://www.youtube.com/user/BinaryTradingVideos' },
                                     { media: 'facebook',  href: 'https://www.facebook.com/derivdotcom' },
                                     { media: 'twitter',   href: 'https://www.twitter.com/derivdotcom' },
-                                    { media: 'telegram',  href: 'https://t.me/binarydotcom' },
+                                    { media: 'telegram',  href: 'https://t.me/derivdotcomofficial' },
```

Comment:
@mitra-deriv Not related to your PR, But instead of maintaining the links in 3 different places we could simplify this by adding a new property to each link to define their visibility 🤔 

```js
{
  media: 'telegram',
  href: 'https://t.me/derivdotcomofficial',
  visibility: ['eu', 'uk']
 }
```

Reply:
@farzin-fs Thanks for the review, You are correct but I'm not sure if we currently do any improvements in the code in binary.com project @habib-deriv please let me know if I am correct else I will create an improvement card to address this comment  🙏